### ISSUE TITLE
Suggestions / Feedback

### ISSUE THREAD

**Issue by webcdn** [vulrun/File-Explorer#1]

**Comment by jsEveryDay** [comment]
Beautifully written. I love how you're calling everything from JS.

1. Filesize ..
You are using a very small % of the material theme (js/css), select only the modules you're using, gulp it and generate a smaller file size.
2. Before you begin working on the folder pane create a "Rows View" with table like viewing of files, the smaller the size of each row the better, the more it can fit in 1 window (eXtplorer like). Enable drag and drop of files into folders.
3. Close modal when user clicks on the background.

Ill get number 2 done and fork it over to you.

**Comment by webcdn** [comment]
> Beautifully written. I love how you're calling everything from JS.
> 
> 1. Filesize ..
>    You are using a very small % of the material theme (js/css), select only the modules you're using, gulp it and generate a smaller file size.
> 2. Before you begin working on the folder pane create a "Rows View" with table like viewing of files, the smaller the size of each row the better, the more it can fit in 1 window (eXtplorer like). Enable drag and drop of files into folders.
> 3. Close modal when user clicks on the background.
> 
> Ill get number 2 done and fork it over to you.

@jsEveryDay Thanks for the appreciation. New Beta Version is available. In this, we have taken care of the point listed.

Suggestions are most welcomed.

**Comment by jsEveryDay** [comment]
All of a sudden the code changed, totally obfuscated.
Not trustworthy, could be a shell.

**Comment by webcdn** [comment]
@jsEveryDay We have not obfuscated the code we just remove line breaks and extra spaces. we did this to reduce the file-size. Looking forward to your suggestions.

**Comment by jsEveryDay** [comment]
@webcdn yeah but you haven't just minified your code, it is also uglified. Wouldn't call it open source.
Regardless, good luck.

**Comment by webcdn** [comment]
@jsEveryDay we really appreciate it and will upload the normal source code in the next build.

**Comment by jsEveryDay** [comment]
@webcdn Cool. Thanks for your prompt responses.
I understand you want to protect your code but what's the point of publishing it. I saw your repo awhile back and it seemed like an interesting project. But I don't understand what use could anyone have for this:
<img width="1280" alt="chrome_2019-03-22_11-31-20" src="https://user-images.githubusercontent.com/16431055/54834501-afd09a80-4c96-11e9-9237-adcc24da2479.png">

Anyways, good work overall. Wish I could jump in on this project.
Next step should be to implement a Dir/File Tree like the good old eXtplorer in my opinion.
I just don't feel comfortable dropping a php file in my server when I see the above.

**Comment by webcdn** [comment]
@jsEveryDay Thanks for the appreciation. Surely, you can contribute to this project. Our main motto is to fuse all codes in a single standalone file as it's in _adminer.php_ (phpMyAdmin alternative).

**What use could anyone have for this?**
>I started this project for my own, but as time float around, my colleagues also demands the same.
So, I decided to put this on GitHub. You can also check to this: [File Explorer](https://www.quora.com/What-is-the-best-program-any-language-youve-ever-written-for-yourself-What-did-it-do-Can-you-share-some-code-for-us-to-admire/answer/Mohit-Gupta-1264)

Regarding the directory/file tree, we know this one is a nice approach and needs a helping hand to do the same.

We will soon upload the main source code with minor bug fixes and improvements.

**Comment by Nerdy-Technician** [comment]
May I ask How do I change the Folder ? im viewing my /var/www/html folder but i wish to show mu /NAS/ folder can you help?

**Comment by webcdn** [comment]
@Roffo66
There are certain cases:
- If your target directory `/NAS/` exists within `/var/www/html` then you can access it using server address `127.0.0.1/NAS/` or `localhost/NAS/` or `{{host}}/NAS/`
- If you are planning to serve another directory, then you need to enable it using Virtual Hosting.

I hope this helps you. Please feel free to drop any message if you have any questions or concerns.

**Comment by Nerdy-Technician** [comment]
How would I go through doing that ?

**Comment by webcdn** [comment]
@Roffo66
If you can explain the scenario this in a bit deeper, then it would help me get to point where you got stuck.

Here are a few questions:
* What's your server address/hostname?
* What's your case in my previous defined cases?

**Comment by Nerdy-Technician** [comment]
Ok so I have a 10TB NAS drive mounted at /NAS/ Linux location so a the folder NAS in root, this using folder location /var/www/html/ I want to view the contents of /NAS/ and not /var/www/html/

**Comment by webcdn** [comment]
> Ok so I have a 10TB NAS drive mounted at /NAS/ Linux location so a the folder NAS in root, this using folder location /var/www/html/ I want to view the contents of /NAS/ and not /var/www/html/

@Roffo66 May I know which server you are using Apache2 / Nginx?

**Comment by Nerdy-Technician** [comment]
I am using Apache 2 up to date

**Comment by webcdn** [comment]
> I am using Apache 2 up to date

* Login as a **root** user, hope you know about `sudo su`
* Go to this file `/etc/apache2/apache2.conf` search for `<Directory /var/www/>` and replace to your directory `<Directory /NAS/>`
* Now, Go to `/etc/apache2/sites-available/000-default.conf` search for DocumentRoot /var/www/html and replace to your DocumentRoot `/NAS`
* Reload / Restart Apache2 service using `service apache2 reload` and `service apache2 restart`

**Comment by 17t104** [comment]
What is tha password

**Comment by webcdn** [comment]
@17t104 When you will load it the first moment. It will prompt you enter the same and then will take you to next world.
By default it was set as `admin`

**Comment by Barracuda1337** [comment]
How can I show pictures instead of icons?
![image](https://user-images.githubusercontent.com/53797515/104965996-f0afac80-59f0-11eb-8bc7-44baf31ca915.png)
I tried but somehow I couldn't get out of it

**Comment by jz0ojiang** [comment]
Hi, I have Sinicized your project. Do I need to submit a PR directly as a new branch or Relase?
https://github.com/0ojixueseno0/File-Explorer
